prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -233,7 +237,13 @@ public async Task SendEnrolleeRenewalEmails()
 
             foreach (var enrollee in enrollees)
             {
+                if (await _enrolleeService.IsEnrolleeAbsentAsync(enrollee.Id))
```

Comment:
is it possible to update the query  in line `225` using a join with the EnrolleeAbsense table to get the enrollees need renewal rather than calling `IsEnrolleeAbsentAsync` in the loop to skip absent enrollees. This could save a lot unnecessary database query in the loop.